app/schemas/selectors/cinematic.js

```diff
@@ -102,49 +188,136 @@ const characterThangTypeSlug = character => {
   if (!character) {
     return
   }
-  if (!character.type || character.type !== 'slug') {
+
+  const thangType = getCharacterFromThangTypeSchema((character || {}).thangType)
+  if (!thangType) {
     return
   }
 
-  if (!character.slug) {
-    throw new Error(`no slug on char`)
+  const { slug, thang } = thangType
+
+  if (typeof character.enterOnStart !== 'boolean') {
```

Comment:
Would it not always be boolean since we have it enforced with the schema?

Reply:
That's true.  I'll remove this check.